tools/project_manager/edit_common.inc

```diff
@@ -94,6 +95,39 @@ function genre_list($genre)
 
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
+function charsuite_list($charsuites, $project_charsuites)
+{
+    $enabled_charsuites = CharSuites::get_enabled();
+
+    $all_charsuites = [];
+    foreach($enabled_charsuites as $global_charsuite)
```

Comment:
We should also rename `$global_charsuite` to `$enabled_charsuite` so it matches the plural version.